docs/user-guide/reduction-workflow-guidelines.md

```diff
@@ -122,6 +122,31 @@ Users should not have to worry about the concrete type of parameters.
   E.g., `sc.scalar(2, unit='m')` and `sc.scalar(2.0, unit='m')` should be usable interchangeably.
   This can also apply to arrays, for instance, `sc.linspace` and `sc.arange` should be interchangeable but the latter may result in integers while the former typically produces floats.
 
+### C.5: Use a conventional pattern minimizing programming jargon, favor fewer and concepts
```

Comment:
Not sure how you mean this sentence. Is it
```suggestion
### C.5: Use a conventional pattern minimizing programming jargon, favor fewer concepts
```
or
```suggestion
### C.5: Use a conventional pattern minimizing programming jargon and concepts and favor fewer concepts
```
?

In any case, I would be more specific:
```suggestion
### C.5: Use a fixed pattern for creating, manipulating, and running workflows
```